src/main/java/com/razorpay/ApiUtils.java

```diff
@@ -15,6 +17,8 @@
 class ApiUtils {
 
   private static OkHttpClient client;
+  private static Map<String, String> headers = new HashMap<String, String>();
+  private static final String version = "1.2.0";
```

Comment:
dont hard code this. Use it from pom.xml. [Check This](http://stackoverflow.com/a/3697482)